src/configs/load.go

```diff
@@ -1,11 +1,23 @@
 package configs
 
 import (
+	"fmt"
+	"os"
+
 	"github.com/ilyakaznacheev/cleanenv"
 )
 
 func LoadEnvs[T any]() (configs T, err error) {
-	err = cleanenv.ReadConfig(".local.env", &configs)
+	mode := os.Getenv("MODE")
```

Comment:
porque não "ENV"?
```suggestion
	mode := os.Getenv("ENV")
```

daí joga um `lower()` e carrega o arquivo

Reply:
done